src/iris/api.py

```diff
@@ -610,6 +610,13 @@ def is_valid_tracking_settings(t, k, tpl):
     else:
         if t not in cache.modes:
             return False, 'Unknown tracking type: %s' % t
+
+    environment = SandboxedEnvironment()
```

Comment:
This should go with the `else` in line 610, otherwise we validate email twice.

Reply:
fixed